docs/design/persistentdisks.md

```diff
@@ -0,0 +1,96 @@
+# Persistent Volumes
+
+This document defines persistent, cluster-scoped storage for applications requiring long lived data.  
+
+
+## Goals
+
+* Allow administrators to create and manage available persistent storage devices
+* Allow developers and pod authors to request storage and 
```

Comment:
Hi Tim.  There is a line in the Constraints section stating that storage shared between pods is out of scope for now.  I'm sure it will be necessary in the future, as GCEPDs can be attached to many hosts as read-only and NFS shares can be read/write across many pods.  We are punting on that for now.